File: app/src/main/java/com/example/shiranpeer/burritolocator/model/Place.java
package com.example.shiranpeer.burritolocator.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Place implements Serializable {
    private String name;
    private String vicinity;
    private String description;
    @SerializedName("price_level")
    private int priceLevel;
    private Geometry geometry;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getVicinity() {
        return vicinity;
    }

    public void setVicinity(String vicinity) {
        this.vicinity = vicinity;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getPriceLevel() {
        return priceLevel;
    }

    public void setPriceLevel(int priceLevel) {
        this.priceLevel = priceLevel;
    }

    public Geometry getGeometry() {
        return geometry;
    }

    public void setGeometry(Geometry geometry) {
        this.geometry = geometry;
    }
}

File: app/src/main/java/com/example/shiranpeer/burritolocator/model/Place.kt
package com.example.shiranpeer.burritolocator.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Place : Serializable {
    var name: String? = null
    var vicinity: String? = null
    var description: String? = null
    @SerializedName("price_level")
    var priceLevel: Int = 0
    var geometry: Geometry? = null
}
